src/components/decimal/decimal.spec.tsx

```diff
@@ -1457,4 +1461,27 @@ describe("Decimal", () => {
       expect(label.prop("isRequired")).toBe(true);
     });
   });
+
+  describe("when maxWidth is passed", () => {
+    it("should be passed to InputPresentation", () => {
+      mount(<Decimal maxWidth="67%" />);
+
+      assertStyleMatch(
+        {
+          maxWidth: "67%",
+        },
+        wrapper.find(InputPresentation)
+      );
+    });
+
+    it("renders correctly as 100% as maxWidth has been left with no value", () => {
```

Comment:
As commented earlier on `input-presentation.spec.tsx`, I would like to change this test description - if you do change it, let's change all of these to match it.